packages/sources/react/src/components/VtmnTextInput/VtmnTextInput.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import clsx from 'clsx';
+
+import '@vtmn/css-text-input';
+
+export interface VtmnTextInputProps
+  extends React.InputHTMLAttributes<HTMLInputElement> {
+  /**
+   * Label text linked to the input
+   * @type {string}
+   */
+  label: string;
+
+  /**
+   * Helper text to help the user
+   * @type {string}
+   */
+  helperText?: string;
+
+  /**
+   * Whether input is successful or not
+   * @type {boolean}
+   */
+  valid?: boolean;
+
+  /**
+   * Whether input is in error or not
+   * @type {boolean}
+   */
+  error?: boolean;
+}
+
+export const VtmnTextInput = ({
+  className,
+  disabled = false,
+  error = false,
+  helperText,
+  id,
+  label,
+  placeholder,
+  valid = false,
+  ...props
+}: VtmnTextInputProps) => {
+  return (
+    <>
+      <label className="vtmn-text-input_label" htmlFor={id}>
+        {label}
+        <input
```

Comment:
Why did you put the `<input>` inside the `<label>`?
See https://github.com/Decathlon/vitamin-web/blob/main/packages/showcases/css/stories/components/text-input/examples/default.html

Reply:
Because i'm used to do that. And i think that's a HTML compliance. (Both are linked with id)
I can move it ;) 
https://developer.mozilla.org/fr/docs/Web/HTML/Element/Label

Reply:
Nope, it's not the best practice.
The "for" already do the job to avoid having input inside label.

See this from W3C:

 ![image](https://user-images.githubusercontent.com/9600228/114980144-51357500-9e8c-11eb-92b7-34899a82b7a4.jpeg)

Reply:
Another important thing to note

```html
<label for="my-input" ...
<input id="my-input"...
```

Is supported by a larger number of screen readers